tools/labelme/labelme.go

```diff
@@ -0,0 +1,50 @@
+package labelme
+
+import (
+	"encoding/json"
+
+	"github.com/linkernetworks/annotation"
+)
+
+type LabelME struct {
+	Shapes    []Shape `bson:"shapes" json:"shapes"`
+	LineColor [4]int  `bson:"lineColor" json:"lineColor"`
+	FillColor [4]int  `bson:"fillColor" json:"fillColor"`
+	ImagePath string  `bson:"imagePath" json:"imagePath"`
+	ImageData string  `bson:"imageData" json:"imageData"`
+}
+
+type Shape struct {
+	Label     string  `bson:"label" json:"label"`
+	LineColor *[4]int `bson:"line_color" json:"line_color"`
+	FillColor *[4]int `bson:"fill_color" json:"fill_color"`
+	Points    []Point `bson:"points" json:"points"`
+}
+
+type Point [2]int
+
+func NewLabelME(imgPath string, label string, lineColor [4]int, fillColor [4]int, imgData []byte) *LabelME {
```

Comment:
when constructing object with more than 3 parameters, we should avoid using constructor function.

Reply:
any documentation to this rule?

Reply:
the book "Clean Code" :-p